app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
```

Comment:
we talked about in class how we can leverage our associations to avoid setting the foreign key manually!